packages/shared/styles/styleUtils/resets/linkReset.ts

```diff
@@ -7,4 +7,8 @@ export const linkReset = css`
   &:visited {
     color: inherit;
   }
+
+  :focus {
```

Comment:
I'm strongly opposed to removing the focus outline on links. We should only hide it in specific cases where we have our own focus styles.

I'll push up a commit with my suggestion for how we should handle this for the sidebar items